Sources/ViewInspector/SwiftUI/ViewThatFits.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+internal extension ViewType {
+    struct ViewThatFits {}
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+extension ViewType.ViewThatFits: SingleViewContent {
+
+    static func child(_ content: Content) throws -> Content {
+        let view: Any = try {
+            guard let rootContent = try? Inspector.attribute(path: "_tree|content", value: content.view) else {
+                // A ViewThatFits View renders only one of its child Views based on the available horizontal space.
```

Comment:
This comment was added to explicitly document that we currently return all children of a `ViewThatFits` View.

cc: @bachand

Reply:
`Inspector.attribute(path: "_tree|content", value: content.view)` is what will return all children right? If so I wonder if this comment should go above that line instead of in the `else` of the `guard` statement.